README.md

```diff
@@ -21,43 +21,57 @@ ember install ember-local-storage-decorator
 Usage
 ------------------------------------------------------------------------------
 
+This addon provides two decorators: `@localStorage` and `@sessionStorage`. 
+Both work identically, the only difference is the backing storage used. 
+`@localStorage` persists data across browser sessions, while `@sessionStorage` 
+only persists data for the duration of the page session.
```

Comment:
I feel we should mention `window.sessionStorage` and `window.localStorage` explicitly to clarify that it's basically a wrapper to interact with both in an Ember application.

Maybe something like this:

> `@localStorage` persists data in [`window.localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) available across browser sessions, while `@sessionStorage`
> persists data in [`window.sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) only for the duration of the current page session.